src/index.js

```diff
@@ -82,23 +87,42 @@ async function run() {
       // skipVersionFile can mean there is no version file and skipCommit can mean that the user
       // is only interested in tags
       if (skipVersionFile || skipCommit) {
-        core.info('Using GIT to determine the new version');
-        const versioning = await handleVersioningByExtension('git', versionFile, versionPath, recommendation.releaseType)
+        core.info('Using GIT to determine the new version')
+        const versioning = await handleVersioningByExtension(
+          'git',
+          versionFile,
+          versionPath,
+          recommendation.releaseType
+        )
         newVersion = versioning.newVersion
-
       } else {
-        const files = versionFile.split(',').map(f => f.trim())
+        const files = versionFile.split(',').map((f) => f.trim())
         core.info(`Files to bump: ${files.join(', ')}`)
 
-        const versioning = await Promise.all(files.map((file) => {
-          const fileExtension = file.split('.').pop()
-          core.info(`Bumping version to file "${file}" with extension "${fileExtension}"`)
-          return handleVersioningByExtension(fileExtension, file, versionPath, recommendation.releaseType)
-        }));
+        const versioning = await Promise.all(
+          files.map((file) => {
+            const fileExtension = file.split('.').pop()
+            core.info(`Bumping version to file "${file}" with extension "${fileExtension}"`)
+            return handleVersioningByExtension(fileExtension, file, versionPath, recommendation.releaseType)
+          })
+        )
 
         newVersion = versioning[0].newVersion
       }
 
+      let gitTag = `${tagPrefix}${newVersion}`
+
+      if (preChangelogGeneration) {
+        const newVersionAndTag = await require(path.resolve(preChangelogGeneration)).preChangelogGeneration({
```

Comment:
Can we change this to also use `process.cwd()`? Then we are inline with the other one, maybe also do `path.resolve` there like here and remove the then un needed import.

Reply:
Sure. I also changed for pre-commit hook by the same logic